TypeCobol/Compiler/CupPreprocessor/CompilerDirectiveBuilder.cs

```diff
@@ -160,7 +160,9 @@ public virtual void EnterCopyCompilerStatement(TypeCobol.Compiler.Scanner.Token
             }
         }
 
-        private static bool BuildReplaceOperation(IList<ReplaceOperation> replaceOperations, ref Token comparisonToken, ref Token[] followingComparisonTokens, ref Token replacementToken, ref Token[] replacementTokens, bool replaceTokens, List<Token> operandTokens)
+        private bool BuildReplaceOperation(IList<ReplaceOperation> replaceOperations, ref Token comparisonToken, ref Token[] followingComparisonTokens, 
```

Comment:
This is getting more and more confusing, the multiple `ref` parameters, the optional `leading` and `trailing` parameters...
I think this should be refactored into a `ReplaceOperationBuilder` class. Also we have now two ways to add errors on REPLACE, the method itself and the caller checking the boolean return value...